basiclibrary/src/test/java/inheritance/TestRestaurant.java

```diff
@@ -0,0 +1,82 @@
+        inputEmployees.add(new Employee("Jim") );
+        inputEmployees.add(new Employee("Michael") );
+        inputEmployees.add(new Employee( "Jane"));
+        Restaurant classForTesting = new Restaurant("The Office Hour", "Scranton, PA", inputEmployees );
+
+        assertEquals("name should match", "The Office Hour", classForTesting.getName());
+        assertEquals("location should match", "Scranton, PA", classForTesting.getLocation() );
+        assertEquals("employee list should match", inputEmployees, classForTesting.getEmployees());
+    }
+
+    @Test
+    public void TestRestaurantToString(){
+        List<Employee> inputEmployees = new ArrayList<>();
+        inputEmployees.add(new Employee("Dwight") );
+        inputEmployees.add(new Employee("Jim") );
+        inputEmployees.add(new Employee("Michael") );
+        inputEmployees.add(new Employee( "Jane"));
+        Restaurant classForTesting = new Restaurant("The Office Hour", "Scranton, PA", inputEmployees );
+        String actualOutput = classForTesting.toString();
+        String expectedOutput = "This restaurant's name is The Office Hour.\n" +
+                "They are located in Scranton, PA,\n" +
+                "and currently have 4 employee(s).\n" +
+                "There are 0 reviews.";
+
+        assertEquals("toString() should output", expectedOutput, actualOutput);
+    }
+
+    @Test
+    public void TestReview(){
+        List<Employee> inputEmployees = new ArrayList<>();
+        inputEmployees.add(new Employee("Dwight") );
+        inputEmployees.add(new Employee("Jim") );
+        inputEmployees.add(new Employee("Michael") );
+        inputEmployees.add(new Employee( "Jane"));
+        Restaurant restaurant1 = new Restaurant("The Office Hour", "Scranton, PA", inputEmployees );
+
+        String userName = "clu";
+        String userReview = "\"Nothing but perfection!\"";
+        int starRating = 5;
+        Review review1 = new Review(userName, userReview, starRating);
+        restaurant1.addReview(review1);
+        String expectedOutput = "clu posted the following review about The Office Hour:\n" +
+                "\"Nothing but perfection!\"";
+        String actualOutput = review1.toString();
+
+        assertEquals("toString() should output", expectedOutput, actualOutput);
+    }
+
+    @Test
+    public void TestRestaurantAndReviewAssociation(){
+        List<Employee> inputEmployees = new ArrayList<>();
+        inputEmployees.add(new Employee("Dwight") );
+        inputEmployees.add(new Employee("Jim") );
+        inputEmployees.add(new Employee("Michael") );
+        inputEmployees.add(new Employee( "Jane"));
+        Restaurant restaurant1 = new Restaurant("The Office Hour", "Scranton, PA", inputEmployees );
+
+        String userName = "clu";
+        String userReview = "\"Nothing but perfection!\"";
+        int starRating = 5;
+        Review review1 = new Review(userName, userReview, starRating);
```

Comment:
it would have been a good idea to create and add more than one review to the restaurant to see if you could add more than one review. It also would've been a good idea to write a test to make sure that you couldn't add the same review to a restaurant again